charts/fleet/values.yaml

```diff
@@ -81,3 +80,6 @@ propagateDebugSettingsToAgents: true
 
 migrations:
   clusterRegistrationCleanup: true
+
+## Additional environment variables to be injected to the fleet-controller container
+additionalEnvs: {}
```

Comment:
I think I would prefer an approach similiar to how [nginx](https://artifacthub.io/packages/helm/bitnami/nginx?modal=values) handles env vars from secrets:

```
## @param extraEnvVarsSecret Secret with extra environment variables
##
extraEnvVarsSecret: ""
```

However, the secret would have to be created manually in this case. Which is not as automatic/user friendly?

Additionally we're adding `extraEnvs`, like Rancher, to allow users to specify non-secret env vars as helm values.